tests/platform_tests/mellanox/check_sysfs.py

```diff
@@ -19,16 +17,11 @@ def check_sysfs(dut):
 
     logging.info("Check content of some key files")
 
-    assert not wait_until_fan_speed_set_to_default(dut, timeout=120), \
-        "Content of /var/run/hw-management/thermal/pwm1 should be 153"
-
-    file_suspend = dut.command("cat /var/run/hw-management/config/suspend")
-    assert file_suspend["stdout"] == "1", "Content of /var/run/hw-management/config/suspend should be 1"
 
     file_asic = dut.command("cat /var/run/hw-management/thermal/asic")
     try:
         asic_temp = float(file_asic["stdout"]) / 1000
-        assert 0 < asic_temp < 85, "Abnormal ASIC temperature: %s" % file_asic["stdout"]
+        assert 0 < asic_temp < 105, "Abnormal ASIC temperature: %s" % file_asic["stdout"]
```

Comment:
The critical threshold for spectrum 1&2 is still 85C, right? Maybe the checking of asic_temp should be based on different spectrum generations?

Reply:
In the Mellanox thermal control algorithm, it defined 105C as "max hot" trip point, it is general for all the ASIC family, here we also use this value to keep align with the algorithm.  It's true that for Spectrum 1 and 2 seldomly can reach "max hot".